library/core/src/array/mod.rs

```diff
@@ -176,6 +176,18 @@ where
     }
 }
 
+#[stable(feature = "try_from_mut_slice_to_array", since = "1.57.0")]
```

Comment:
```suggestion
#[stable(feature = "try_from_mut_slice_to_array", since = "1.58.0")]
```

Reply:
Thanks! I applied the suggestion at 66e0523.